tests/playwright-test/playwright.expect.misc.spec.ts

```diff
@@ -262,6 +262,56 @@ test('should support toHaveClass w/ array', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(1);
 });
 
+test('should support toContainClass', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.test.ts': `
+      const { test } = pwt;
+
+      test('pass', async ({ page }) => {
+        await page.setContent(\`
+          <div class="foo bar baz">
+            <span class="alice item baz"></span>
+            <span class="bob item baz"></span>
+          </div>
+        \`);
+        const locator = page.locator('div');
+        await expect(locator).toContainClass('foo');
+        await expect(locator).toContainClass('bar');
+        await expect(locator).toContainClass('baz');
+        await expect(locator).toContainClass('foo baz');
+        await expect(locator).toContainClass('baz foo');
```

Comment:
- Add a test with leading/trailing whitespace in `expected`.
- Add a test with empty `expected`.